master/master.cfg

```
@@ -419,13 +419,14 @@ for git_url, branchname, git_branch in git_branches:
         builderNames=buildernames,
         fileIsImportant=is_important_change,
     ))
-    c['schedulers'].append(Nightly(
-        name=branchname + '-daily',
-        hour=0,
-        minute=0,
-        change_filter=util.ChangeFilter(branch=git_branch),
-        builderNames=dailybuildernames,
-    ))
+    if dailybuildernames:
+        c['schedulers'].append(Nightly(
+            name=branchname + '-daily',
+            hour=int(24 / (len(git_branches) - 1) * branch_num),
```

Hum, do we get hour=24 for the last Git branch? Should we limit to hour=23?

```suggestion
            hour=int(branch_num / (len(git_branches) - 1) * 23),
```